src/e2e-test/features/dataplex/source/DesignTimeDataplex.feature

```diff
@@ -0,0 +1,95 @@
+
+  @BATCH-TS-DATAPLEX-DSGN-01
+  Scenario: To validate data when pipeline is created between DataPlex to DataPlex
+    When Open Datafusion Project to configure pipeline
+    Then Select plugin: "Dataplex" from the plugins list as: "Source"
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Connect plugins: "Dataplex" and "Dataplex2" to establish connection
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexEntityID" with value: "dataplex.entity"
+    And Click on the Get Schema button
+    And Validate output schema with expectedSchema "dataplex.SourceSchema"
+    And Validate "Dataplex" plugin properties
+    And Close the Plugin Properties page
+    And Navigate to the properties page of plugin: "Dataplex2"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexAssetID" with value: "dataplex.assetBQ"
+    And Enter input plugin property: "dataplexTableName" with value: "dataplex.outputTable"
+    And Validate "Dataplex" plugin properties
+    And Close the Plugin Properties page
+
+  @BATCH-TS-DATAPLEX-DSGN-02
+  Scenario: To verify user should be able to validate output schema successfully in dataplex source
+    When Open Datafusion Project to configure pipeline
+    Then Select plugin: "Dataplex" from the plugins list as: "Source"
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Connect plugins: "Dataplex" and "Dataplex2" to establish connection
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexEntityID" with value: "dataplex.entity"
+    And Click on the Get Schema button
```

Comment:
Looks like we aren't actually validating the schema? Same thing in other tests below.

Reply:
Added the steps to validate the output schema.